dts/arm/nxp/nxp_imx95_m7.dtsi

```diff
@@ -0,0 +1,154 @@
+/*
+ * Copyright 2024 NXP
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <arm/armv7-m.dtsi>
+#include <zephyr/dt-bindings/clock/imx95_clock.h>
+#include <mem.h>
+
+/ {
+	cpus {
+		#address-cells = <1>;
+		#size-cells = <0>;
+
+		cpu@0 {
+			device_type = "cpu";
+			compatible = "arm,cortex-m7";
+			reg = <0>;
+
+			#address-cells = <1>;
+			#size-cells = <1>;
+
+			mpu: mpu@e000ed90 {
+				compatible = "arm,armv7m-mpu";
+				reg = <0xe000ed90 0x40>;
+			};
+		};
+	};
+
+	reserved-memory {
```

Comment:
suggestion: drop this node. It has no real meaning for us. It was originally added to mirror Linux DT. I'd also suggest renaming `scmi_res0` to `scmi_shmem0` or something like that.

Reply:
I think we could keep to use reserve memory, so it is clear that these memory regions are reserved common memory, but not hardware module's memory address, although it has no explicit function, but make it is readable.